app/views/hackathon_mailer/swag_request.text.erb

```diff
@@ -0,0 +1,9 @@
+Hey mail team! 👋
+
+Hackathon swag has been requested for:
+
+<%= @hackathon.applicant.name %> c/o <%= @hackathon.name %>
+<%= @mailing_address.line1 %>
+<%= @mailing_address.line2 %>
+<%= @mailing_address.city %><%= ", #{@mailing_address.province} #{@mailing_address.postal_code}" if @mailing_address.province %>
```

Comment:
```suggestion
<% city, *province_and_postal = [@mailing_address.city, @mailing_address.province, @mailing_address.postal_code].compact_blank %>
<%= [city, province_and_postal.join(" ")].compact_blank.join(", ") %>
```

Reply:
What's the point of this?

Reply:
it handles the case where _any_ three of them are nil/blank. we can have addresses with a missing province or postal code!